src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_app_extension.cc

```diff
@@ -123,7 +125,43 @@ void RCAppExtension::SaveResumptionData(
 
 void RCAppExtension::ProcessResumption(
     const smart_objects::SmartObject& saved_app,
-    resumption::Subscriber subscriber) {}
+    resumption::Subscriber subscriber) {
+  LOG4CXX_AUTO_TRACE(logger_);
+
+  if (!saved_app.keyExists(
+          application_manager::strings::application_subscriptions)) {
+    LOG4CXX_DEBUG(logger_, "application_subscriptions section is not exists");
```

Comment:
```suggestion
    LOG4CXX_DEBUG(logger_, "application_subscriptions section does not exist");
```

Reply:
@LitvinenkoIra 
Thanks, fixed in:https://github.com/LuxoftSDL/sdl_core/pull/99/commits/114a2b4f4858dd189f26c13f00aa4292489b374b